provider/postgis/postgis.go

```diff
@@ -126,7 +127,10 @@ type Provider struct {
 	queryHistogramSeconds            *prometheus.HistogramVec
 }
 
-func (p *Provider) Collectors(prefix string, cfgFn func(configKey string) map[string]interface{}) ([]observability.Collector, error) {
+func (p *Provider) Collectors(
+	prefix string,
+	cfgFn func(configKey string) map[string]interface{},
```

Comment:
Let's change `interface{}` -> `any`

Reply:
I know you marked this as resolved, but i'm still seeing `map[string]interface{}`?

Reply:
fixed